base/deprecated.jl

```diff
@@ -672,65 +661,6 @@ end
 # Deprecate isimag (#19947).
 @deprecate isimag(z::Number) iszero(real(z))
 
-@deprecate airy(z::Number) airyai(z)
-@deprecate airyx(z::Number) airyaix(z)
-@deprecate airyprime(z::Number) airyaiprime(z)
```

Comment:
How should I deprecate them if the functions they should be deprecated in favor of are no longer in Base? Something like
```julia
function airy(z::Number)
    error("airy(z) has been deprecated in favor of airyai(z) in the SpecialFunctions package.")
end
```
?